contracts/StorageManager.sol

```diff
@@ -129,7 +129,7 @@ contract StorageManager is Ownable, Pausable {
     @notice stops the Offer. It sets the totalCapacity to 0 which indicates terminated Offer.
     @dev no new Agreement can be created and no existing Agreement can be prolonged. All existing Agreement are still valid for the amount of periods still deposited.
     */
-    function terminateOffer() public {
+    function terminateOffer() public whenNotPaused {
```

Comment:
I would not put it here. I think we should allow providers to `cleanup` and make sure they can disable future agreements when contract is resumed. Would give them more safety, on the same lines as allowing all parties to withdraw funds even if paused.

Reply:
Done